src/MF_Output/MFOutput.cpp

```diff
@@ -14,16 +14,28 @@ MFOutput::MFOutput()
 void MFOutput::attach(uint8_t pin)
 {
     _pin   = pin;
-#if !defined(ARDUINO_ARCH_RP2040)
+#if defined(ARDUINO_ARCH_RP2040)
+    pinMode(_pin, OUTPUT_12MA);
+    digitalWrite(_pin, LOW)
+#else
     pinMode(_pin, OUTPUT);
-    set(_value);
+    analogWrite(pin, LOW);
 #endif
 }
 
 void MFOutput::set(uint8_t value)
 {
     _value = value;
+#if defined(ARDUINO_ARCH_RP2040)
```

Comment:
i think it would be easier to read if we introduce a deviated method like

`setRp2040` and then put the custom logic there instead of having everything inline

Reply:
Hmhm, in this case the `CommandMessenger.cpp` must also be changed:
```
#if defined(ARDUINO_ARCH_RP2040)
    cmdMessenger.attach(kSetPin, Output::OnSetRP2040);
#else
    cmdMessenger.attach(kSetPin, Output::OnSet);
#endif
```

Due to the differnet pinMode initialization `MFOutput.cpp` would look like:
```
void MFOutput::attach(uint8_t pin)
{
    _pin   = pin;
    pinMode(_pin, OUTPUT);
    analogWrite(pin, LOW);
}

#if defined(ARDUINO_ARCH_RP2040)
void MFOutput::attachRP2040(uint8_t pin)
{
    _pin   = pin;
    pinMode(_pin, OUTPUT_12MA);
    digitalWrite(_pin, LOW);
}
#endif
```

And `Output.cpp` must be as:
```
#if defined(ARDUINO_ARCH_RP2040)
        outputs[outputsRegistered].attachRP2040(pin);
#else
        outputs[outputsRegistered].attach(pin);
#endif
```

Not sure if this increases readability, also as the `CommandMessegner.cpp` must also be changed (changes not as much bundled as before).

Reply:
i was thinking of a private local method only - nothing fancy

Reply:
After some more testing the logic from Pico to set outputs can also be used for AVR's.
So I wouldn't differ anymore between both. Just for setting `pinMode()` it has to be differ betwenn AVR and Pico.

The class `MFOutput()` is "only" needed for setting the pinMode and setting the powersave Mode.
I am wondering if we still need this class, both could also be done within `Output.cpp` like setting the output.